spring-cloud-config-client/src/main/java/org/springframework/cloud/config/client/aot/ConfigClientHints.java

```diff
@@ -0,0 +1,66 @@
+package org.springframework.cloud.config.client.aot;
+
+import org.springframework.aot.hint.MemberCategory;
+import org.springframework.aot.hint.RuntimeHints;
+import org.springframework.aot.hint.RuntimeHintsRegistrar;
+import org.springframework.aot.hint.TypeReference;
+import org.springframework.boot.context.config.ConfigDataLocation;
+import org.springframework.cloud.config.client.ConfigClientAutoConfiguration;
+import org.springframework.cloud.config.client.RetryTemplateFactory;
+import org.springframework.cloud.config.environment.Environment;
+import org.springframework.cloud.config.environment.PropertySource;
+import org.springframework.retry.support.RetryTemplate;
+import org.springframework.util.ClassUtils;
+
+/**
+ * The config client runtime hints to enable the client to be used for aot builds.
+ *
+ * @author Olga Maciaszek-Sharma
+ * @author Tobias Soloschenko
+ */
+public class ConfigClientHints implements RuntimeHintsRegistrar {
```

Comment:
Please test if it has to be `public` and, if not, change to package access.

Reply:
Changed the access to package access